### PR TITLE
fixed meditation page and added different styles and did journal page 

### PR DESCRIPTION
**Pull request by kirustar14**
added new meditation styles, and journal icon on home page

### CODE REVIEW

File: client/src/components/elements/Journal.tsx
Comment (Shuzhengz) [inline client/src/components/elements/Journal.tsx:1]: im pretty sure we're planning on overhauling the home page in #59 so look out for that

File: client/src/components/pages/journal.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+          >
+            {loading ? 'Saving...' : 'Save Entry'}
+          </button>
+        </div>
+      )}
+
+      {activeTab === 'chart' && (
+        <div className="mood-chart">
+          <h3>Mood Distribution</h3>
+          <Pie data={moodData} />
```
Comment (Shuzhengz) [inline client/src/components/pages/journal.tsx:141]: the pie chart is comically large loll, and the legend is very tiny

image

File: client/src/components/pages/meditation.tsx [hunk 2]
```diff
@@ -1,168 +1,159 @@
...
+      { title: "Notice the Calm", content: "As you breathe, notice the sense of calmness settling over you with each inhale and exhale." },
+      { title: "Be Present", content: "Inhale deeply, and exhale fully. Allow yourself to be fully present in this moment." },
+      { title: "Feel Peaceful", content: "With every breath, feel a growing sense of peace filling your body and mind." },
+    ],
+    "positive-energy-boost": [
+      { title: "Feel the Energy", content: "Take a deep breath and imagine a bright, vibrant energy filling your body." },
+      { title: "Activate Your Mind", content: "Visualize your mind being filled with positive thoughts and energy." },
+      { title: "Energize Your Body", content: "Focus on each body part, feeling a surge of energy revitalizing you." },
+      { title: "Embrace Positivity", content: "Visualize a light surrounding you, pushing out negativity and replacing it with vibrant positivity." },
+      { title: "Feel Empowered", content: "You are full of energy and confidence. Embrace the positivity within you." },
```
Comment (Shuzhengz) [inline client/src/components/pages/meditation.tsx:64]: The text is very hard to see here, maybe change the color

image

File: client/src/components/pages/meditation.tsx [hunk 3]
```diff
@@ -1,168 +1,159 @@
...
+    if (theme && meditationThemes[theme]) {
+      if (direction === "next") {
+        if (step < meditationThemes[theme].length) {
+          setStep((prevStep) => prevStep + 1);
+        }
+      } else if (direction === "prev") {
+        if (step > 0) {
+          setStep((prevStep) => prevStep - 1);
+        }
+      }
```
Comment (Shuzhengz) [inline client/src/components/pages/meditation.tsx:102]: clicking on the previous button on the 0th step probably should bring the user back to the main meditation page, currently it doesn't do anything

image

File: client/src/components/pages/journal.tsx
Comment (Shuzhengz) [inline client/src/components/pages/journal.tsx:1]: nice job on reading and saving data with the server

File: client/src/components/pages/meditation.tsx [hunk 4]
```diff
@@ -1,168 +1,159 @@
...
-      </div>
+      {theme && meditationThemes[theme] && (
+        <div className="meditation-content">
+          {step < meditationThemes[theme].length ? (
+            <>
+              <h2 style={{ color: "white", fontSize: "2rem"  }}>{meditationThemes[theme][step].title}</h2>
+              <p style={{ color: "white", fontSize: "1.5rem" }}>{meditationThemes[theme][step].content}</p>
+              <div className="navigation">
+                <button onClick={() => handleNavigation("prev")}>Previous</button>
+                <button onClick={() => handleNavigation("next")}>Next</button>
```
Comment (Shuzhengz) [inline client/src/components/pages/meditation.tsx:143]: I added material UI as a dependency in #55, so u can use [these](https://mui.com/material-ui/react-button/) once it's merged

File: server/src/index.ts [hunk 5]
```diff
@@ -65,6 +66,8 @@ app.use(express.json());
       process.exit(0);
     });
 
+    */
+
```
Comment (Shuzhengz) [inline server/src/index.ts:70]: a bit confused by this

**Review by Shuzhengz — CHANGES_REQUESTED** [review]
There are some small issues that I noticed, but overall very nice!

File: client/src/components/elements/Journal.tsx
Comment (kirustar14) [reply client/src/components/elements/Journal.tsx:1]: sounds good

File: client/src/components/pages/journal.tsx (hunk 1 above)
Comment (kirustar14) [reply client/src/components/pages/journal.tsx]: fixed lol

Screenshot 2024-12-04 at 8 26 04 AM

File: client/src/components/pages/meditation.tsx (hunk 2 above)
Comment (kirustar14) [reply client/src/components/pages/meditation.tsx:64]: fixed


Screenshot 2024-12-04 at 8 29 41 AM

File: client/src/components/pages/meditation.tsx (hunk 4 above)
Comment (kirustar14) [reply client/src/components/pages/meditation.tsx:147]: sounds good

File: client/src/components/pages/meditation.tsx (hunk 3 above)
Comment (kirustar14) [reply client/src/components/pages/meditation.tsx]: fixed!

File: client/src/components/pages/journal.tsx
Comment (kirustar14) [reply client/src/components/pages/journal.tsx:1]: thank you

**Review by kirustar14 — COMMENTED** [review]
fixed all current issues

**Review by Shuzhengz — APPROVED** [review]
lgtm!

**Review by jut008 — APPROVED** [review]
lgtm

**Pull request merged**